docs/source/install/overview.rst

```diff
@@ -0,0 +1,62 @@
+Overview: Single-box Reference Deployment
+=========================================
+
+.. todo:: short intro paragraph: this is reference deployment for single box, follow installation
+    guide to get here.
+
+.. figure:: https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/pub?w=960&amp;h=720
```

Comment:
Nobody wants to discuss architecture here and nobody says we should hide anything from users. :) It's just that St2 looks a bit messy in this pic, that's all I'm saying, and I wonder if we can make the pic better to turn it into a better first impression. Making St2 better goes into other discussions. :)